lib/Entity/Module.php

```diff
@@ -155,6 +155,10 @@ public function validate()
     {
         if (!v::intType()->validate($this->defaultDuration))
             throw new InvalidArgumentException(__('Default Duration is a required field.'), 'defaultDuration');
+
+        if (!v::alnum(',')->validate($this->validExtensions)) {
```

Comment:
Does this work if its empty?

Reply:
no, it does not...

Do we want something like that? 🤔 

```
        if (v::notEmpty()->validate($this->validExtensions)) {
            if (!v::alnum(',')->validate($this->validExtensions)) {
                throw new InvalidArgumentException(__('Comma separated file extensions only please, without the .'),'validExtensions');
            }
        }
```

Reply:
```
 if (!empty($this->validExtensions) && !v::alnum(',')->validate($this->validExtensions)) {
            throw new InvalidArgumentException(__('Comma separated file extensions only please, without the .'), 'validExtensions');
        }
```

Did you mean that?